tests/rt_cascading.erl

```diff
@@ -199,7 +214,7 @@ big_circle_test_() ->
             end || Node <- Nodes]
         end},
 
-        {"2 way repl, and circle it", timeout, timeout(65), fun() ->
+        {"2 way repl, and circle it", fun() ->
             ConnectTo = ["6", "1", "2", "3", "4", "5"],
             Connect = fun({Node, ConnectToName}) ->
                 Nth = list_to_integer(ConnectToName),
```

Comment:
This fun should be renamed to ConnectNodes and the lists:map should be lists:foreach